data/sql/updates/pending_db_world/rev_1649856189895360036.sql

```diff
@@ -0,0 +1,25 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1649856189895360036');
+
```

Comment:
can u remove the version_db_world line please? there was a change a few days ago and we no longer require this line.